tool/generator/main.dart

```diff
@@ -42,6 +42,13 @@ void main() async {
   // Delete Android's Organization Folder Hierarchy
   Directory(_orgPath).deleteSync(recursive: true);
 
+  // Delete ignored files
+  Directory(path.join(_targetPath, 'my_app'))
+      .listSync(recursive: true)
+      .whereType<File>()
+      .where((file) => file.isIgnored())
+      .forEach((file) => file.deleteSync());
```

Comment:
@renancaraujo would this still generate the files when using the brick generated by this generator?

Reply:
I have no idea what you mean